public/app/features/transformers/configFromQuery/ConfigFromQueryTransformerEditor.test.tsx

```diff
@@ -38,7 +38,7 @@ describe('ConfigFromQueryTransformerEditor', () => {
   it('Should be able to select config frame by refId', async () => {
     setup();
 
-    let select = (await screen.findByText('Config query')).nextSibling!.firstChild!;
+    let select = (await screen.findByText('Config query')).nextSibling!.firstChild!.firstChild!;
```

Comment:
Similar to this comment: https://github.com/grafana/grafana/pull/43706/files#r781104121